packages/vuetify/src/composables/border-radius.ts

```diff
@@ -26,14 +29,15 @@ export const makeBorderRadiusProps = propsFactory({
   rounded: {
     type: [Boolean, Number, String],
     validator: (v: any) => radius.includes(v),
-  },
+  } as Prop<IRounded>,
 })
 
-// Effect
-export function useBorderRadiusClasses (props: BorderRadiusProps) {
-  const borderRadiusClasses = computed(() => {
+export const borderRadiusClassNames = (
```

Comment:
I don't think this needs to be a separate function. Can just move into the code block on line 62

Reply:
Sure.

But I wanted to point out here that it's always cheaper to run test without the framework.  In Vue, and Vue 3 it's cheap. But if we introduce this here, later on, if things gets heavier, we won't have to change the whole code base.

This is from a person who was in an heavy code base where every test were also "testing the framework".